--- pirut-1.3.31/data/pirut.desktop.in~	2007-11-07 00:51:40.000000000 +0200
+++ pirut-1.3.31/data/pirut.desktop.in	2015-05-31 18:05:53.659327790 +0300
@@ -3,8 +3,9 @@
 _GenericName=Add/Remove Software
 _Comment=Change the software installed on the system
 Categories=System;X-Red-Hat-Base;X-Fedora;PackageManager;GNOME;GTK;
-Exec=/usr/bin/getproxy /usr/bin/pirut
+Categories=Application;System;PackageManager;GNOME;GTK
+Exec=gksu -u root /usr/sbin/pirut
 Terminal=false
 Type=Application
 Icon=system-software-installer
 StartupNotify=true
--- pirut-1.3.31/data/pup.desktop.in~	2007-11-07 00:51:40.000000000 +0200
+++ pirut-1.3.31/data/pup.desktop.in	2015-05-31 18:06:33.148048022 +0300
@@ -2,9 +2,9 @@
 _Name=Software Updater
 _GenericName=Software Updater
 _Comment=Update software installed on the system
-Categories=System;X-Red-Hat-Base;X-Fedora;PackageManager;GNOME;GTK;
-Exec=/usr/bin/getproxy /usr/bin/pup
+Categories=PackageManager;GNOME;GTK;System;Application;
+Exec=gksu -u root /usr/sbin/pup
 Terminal=false
 Type=Application
 Icon=system-software-update
 StartupNotify=true
--- pirut-1.3.31/data/system-install-packages.desktop.in~	2007-11-07 00:51:40.000000000 +0200
+++ pirut-1.3.31/data/system-install-packages.desktop.in	2015-05-31 18:07:18.490405998 +0300
@@ -2,11 +2,11 @@
 _Name=Software Installer
 _GenericName=Software Installer
 _Comment=Install selected software on the system
-Categories=System;X-Red-Hat-Base;X-Fedora;
-Exec=/usr/bin/getproxy /usr/bin/system-install-packages %F
+Categories=GTK;System;Application;
+Exec=gksu -u root /usr/sbin/system-install-packages %F
 Terminal=false
 Type=Application
 Icon=system-software-installer
 StartupNotify=true
 NoDisplay=true
 MimeType=application/x-rpm;
