--- pirut-1.3.31/data/Makefile.am~	2008-01-31 18:29:21.000000000 +0200
+++ pirut-1.3.31/data/Makefile.am	2015-05-31 18:12:03.888581130 +0300
@@ -62,10 +62,6 @@
 	$(INSTALL_DATA) -D pup.console $(DESTDIR)$(sysconfdir)/security/console.apps/pup
 	$(INSTALL_DATA) -D system-install-packages.console $(DESTDIR)$(sysconfdir)/security/console.apps/system-install-packages
 	$(INSTALL_DATA) -D system-cdinstall-helper.console $(DESTDIR)$(sysconfdir)/security/console.apps/system-cdinstall-helper
-	$(LN_S) consolehelper $(DESTDIR)$(bindir)/pirut
-	$(LN_S) consolehelper $(DESTDIR)$(bindir)/pup
-	$(LN_S) consolehelper $(DESTDIR)$(bindir)/system-install-packages
-	$(LN_S) consolehelper $(DESTDIR)$(bindir)/system-cdinstall-helper
 
 uninstall-local:
 	rm -f $(DESTDIR)$(sysconfdir)/pam.d/pirut
